src/ServiceControl/InternalsVisibleTo.cs

```diff
@@ -3,4 +3,8 @@
 [assembly: InternalsVisibleTo("ServiceControl.UnitTests")]
 [assembly: InternalsVisibleTo("ServiceControl.AcceptanceTests")]
 [assembly: InternalsVisibleTo("ServiceControl.AcceptanceTesting")]
+[assembly: InternalsVisibleTo("ServiceControl.PersistenceTests")]
 [assembly: InternalsVisibleTo("ServiceControl.MultiInstance.AcceptanceTests")]
+[assembly: InternalsVisibleTo("ServiceControl.Persistence.InMemory")]
+[assembly: InternalsVisibleTo("ServiceControl.Persistence.RavenDb")]
+[assembly: InternalsVisibleTo("ServiceControl.Persistence.SqlServer")]
```

Comment:
Should we just make the things these rely upon public instead?

Reply:
I think I tried that originally but then the classes it was using also had to be changed, and then the classes those were using - I was going down a rabbit hole so I aborted.